### DISCUSSION TITLE
HC_Preface/join/

### DISCUSSION THREAD

**Discussion by giscus** [ProbiusOfficial/Hello-CTF#88] [Show and tell]
# HC_Preface/join/

开源免费、新手友好的CTF(Capture The Flag,夺旗赛)入门教程

https://hello-ctf.com/HC_Preface/join/

**Comment by Strawberry47** [comment]
第一次听说MkDocs-Material，可以将markdown文件转换为静态网页。新知识+1

**Reply by Randark-JMT** [reply]
那 Hexo 和 Docusaurus 怎么说呢🤔
